packages/api/src/controllers/stream.ts

```diff
@@ -1327,6 +1327,83 @@ app.patch(
   }
 );
 
+app.post(
+  "/:id/multistream",
```

Comment:
The path should have a clear name of what **action** is being taken on this API, while "multistream" does not mean much, it's only the name of the field. I'd rename this to either `add-multistream` or even `add-multistream-target`